### ISSUE TITLE
Question on the MSE loss of A2C

### ISSUE THREAD

**Issue by stephenllh** [DLR-RM/stable-baselines3#626]
Hi. One quick question, why is the `values` output by the value network act as the "target" of the MSE loss, instead of the "input"?

```
value_loss = F.mse_loss(rollout_data.returns, values)
```
The code referenced is [here](https://github.com/DLR-RM/stable-baselines3/blob/7b977d7b0344f2828c5af863814fc82868a8af2f/stable_baselines3/a2c/a2c.py#L149).
